src/modules/kvs/kvstxn.c

```diff
@@ -76,6 +78,8 @@ static void kvstxn_destroy (kvstxn_t *kt)
         json_decref (kt->keys);
         json_decref (kt->names);
         json_decref (kt->rootcpy);
+        if (kt->entry)
```

Comment:
cache_entry_decref() is a no-op if entry=NULL so extra check is not needed here.